.github/workflows/wda-package.yml

```diff
@@ -3,13 +3,27 @@ name: Building WebDriverAgent
 on:
   workflow_dispatch:
 
+env:
+  XCODE_VERSION: 14.2
+  DESTI_IOS_SIM: platform=iOS Simulator,name=iPhone 14 Pro
+  DESTI_TVOS_SIM: platform=tvOS Simulator,name=Apple TV
+
 jobs:
-  build:
+  host_machine:
+    runs-on: ubuntu-latest
+    outputs:
+      host: ${{ steps.macos_host.outputs.host }}
+    steps:
+      - run: |
+          echo "host=macos-12" >> $GITHUB_OUTPUT
+        id: macos_host
+  
+  for_real_devices:
```

Comment:
so no way to make it to a matrix?

Reply:
Currently I wonder if we could do something with `needs`, but thinking to tune more with Scripts/build.sh modification